About.html

```diff
@@ -4,17 +4,147 @@
+             </ul>
+             <button type="button" class="ml-28 border-4 border-brandorange text-brandorange p-3">CC Campaign</button>
+          </div>
+        </nav>
     </header>
     <main>
-        <section>
-            <h1>About</h1>
+        <section class="main-head bg-brandgray bg-opacity-40 md:h-full pb-8 md:pb-2">
+            <div class="bg-[url('images/backpattern.png')] bg-no-repeat bg-contain bg-center">
+                <div class="md:hidden">
+                    <img src="images/more.png" class="h-14 p-3" alt="humberger menu" onclick="display()" id="menu">
+                    <img
+                    src="images/close.png"
+                    alt="menu"
+                    id="menu2"
+                    onclick="display()"
+                    class="h-12 p-3"
+                  />
+                </div>
+                <div class="mob-nav-container" id="mnc">
+                    <ul class="mob-nav text-3xl font-black ml-20 text-brandbrown">
+                      <li>
+                        <a href="index.html" onclick="menuClicked()">Home</a>
+                      </li>
+                      <li>
+                        <a href="#" onclick="menuClicked()">Join</a>
+                      </li>
+                      <li>
+                        <a href="#" onclick="menuClicked()">Sponsor</a>
+                      </li>
+                      <li>
+                        <a href="#" onclick="menuClicked()">News</a>
+                      </li>
+                    </ul>
+                  </div>
+            <div class="pt-16 flex flex-col items-center text-center">
+                <h2 class="text-2xl md:text-4xl font-normal text-brandorange md:hidden">Hello! Sharing Cardano</h2>
+               <h1 class="md:w-1/3 text-4xl md:text-5xl text-brandorange font-black md:mt-24 md:mb-5">GLOBAL CARDANO STUDENT SEMINAR 2022</h1>
+               <div class="flex justify-center mt-2 md:mb-5">
+                <div class="bg-brandorange h-0.5 w-6"></div>
+            </div>
+               <div class="md:w-1/3 md:p-5 head-qout w-80 text-sm mt-5 bg-white p-2 md:mb-5">
+                <p class="md:hidden block">Presentations, and demos from Cardano leaders and luminaries hosted in the biggest virtual world ever for a blockchain event. Presentations, and demos from Cardano leaders and luminaries hosted in the biggest virtual world ever for a blockchain event. </p>   
+                <p class="md:black hidden">Presentations, and demos from Cardano leaders and luminaries hosted in the biggest virtual world ever for a blockchain event. Presentations, and demos from Cardano leaders and luminaries hosted in the biggest virtual world ever for a blockchain event. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Labore animi, quos accusantium doloremque soluta perferendis corrupti! Hic repellat reprehenderit voluptatum harum cupiditate. Aut facere rerum officia tenetur minus cupiditate tempora.</p>
```

Comment:
- [ ] Kindly here the paragraph doesn't appear, kindly fix it to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/168819035-d725f943-9549-49ad-a692-151b814492ec.png)